src/lib/rest/restReply.cpp

```diff
@@ -162,8 +162,8 @@ void restReply(ConnectionInfo* ciP, const std::string& _answer)
     }
   }
 
-  //Check if CORS is enabled
-  if (strlen(restAllowedOrigin) > 0)
+  //Check if CORS is enabled and the response is not bad verb response
+  if (strlen(restAllowedOrigin) > 0 && ciP->httpStatusCode != SccBadVerb)
```

Comment:
Suggested names:

* `allowedOrigin` for the string
* `hasAllowedOrigin` for the bool

Reply:
Agreed, this would be much better, thanks for the suggestion. 

One comment about the name of the boolean variable. In the code we basically do the check (the length check as of now) to see if CORS is enabled or not. I think it would make more sense to have the variable name something like `corsEnabled`. This would also make the code easier to understand. 

What do you think @kzangeli @fgalan ?

Reply:
`corsEnabled` sounds great

Reply:
`corsEnabled` is fine for the booll

Reply:
Fixed in b9a8b2f